src/css/styles.css

```diff
@@ -2,6 +2,124 @@
 @import './global.css';
 
 body {
-  font-family: 'Nunito', sans-serif; /* Using Nunito font family*/
-  background-color: var(--primary-color); /*using a css variable*/
+  background-color: var(--light-blue-color);
+}
+
```

Comment:
we could add a comment referencing the section of the code where we use those styles, something like 
```
/*--------------------------------
  --------- MEALS STYLES -------
  --------------------------------*/
```